extensions/src/main/java/org/apache/falcon/extensions/store/ExtensionStore.java

```diff
@@ -218,17 +229,24 @@ public String getExtensionLibPath(final String extensionName) throws StoreAccess
         }
     }
 
-    public String getExtensionResource(final String resourcePath) throws StoreAccessException {
+    public String getExtensionResource(final String resourcePath) throws FalconException {
```

Comment:
This method is used only by trusted extension. Custom extensions won't be using this. Hence no change required. However it is good to keep it generic.